src/prebid.js

```diff
@@ -301,7 +301,7 @@ $$PREBID_GLOBAL$$.renderAd = function (doc, id) {
         var url = adObject.adUrl;
         var ad = adObject.ad;
 
-        if (doc === document || adObject.mediaType === 'video') {
+        if (!utils.inIframe() || adObject.mediaType === 'video') {
```

Comment:
Unfortunately this logic is wrong, it fails for the standard prebid case (always evaluates to true when not in an iframe).